### PR TITLE
Adjust Root to be Reminders

### PR DESCRIPTION
**Pull request by marissa27**
Added in an index.js to route
Redirected homepage ('/') to ('/reminders')
Rewrote test for homepage to make sure it reroutes to /reminders

## Purpose

To redirect root of the app to /reminders. Should still see all reminders on the page.

## Approach

Look back at ember tutorial then went into ember docs.

### Learning

Learning about model transitioning using the beforeModel()

(Ember Doc)[https://guides.emberjs.com/v2.12.0/routing/redirection/]
(Ember Tutorial)[https://guides.emberjs.com/v2.12.0/tutorial/routes-and-templates/]

### Test coverage 

We rewrote initial test for root url to encompass the new redirect for root to go to /reminders. 

### Follow-up tasks

Close Issue #2 
Will start: 
[Issue #3]

#### After
Imgur

### CODE REVIEW

**Comment by marissa27** [comment]
@martensonbj @Tman22

File: app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel() {
+   this.replaceWith('reminders');
+ }
```
Comment (martensonbj) [inline app/routes/index.js]: Match up the indentation here with the beforeModel hook.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Tiny clean up request

**Comment by martensonbj** [comment]
@marissa27 @Tmee status on this small change?

**Comment by martensonbj** [comment]
👍 Thanks

**Pull request merged**